ginga/gtk3w/GtkHelp.py

```diff
@@ -401,9 +401,9 @@ def append_page(self, widget, label):
         # pick a random spot to place the window initially
         rect = self.get_allocation()
         wd, ht = rect.width, rect.height
-        x = random.randint(self.cascade_offset,
+        x = random.randint(self.cascade_offset,  # nosec
```

Comment:
Hmm, it flags use of `random` module due to seed not being truly random enough?

Reply:
No, it's actually not that relevant unless we doing crypto. See https://bandit.readthedocs.io/en/latest/blacklists/blacklist_calls.html#b311-random